packages/lambdas/src/shared/patient-import.ts

```diff
@@ -32,28 +23,40 @@ export function parseFacilityId(bodyAsJson: any) {
 //eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function parseTriggerConsolidated(bodyAsJson: any) {
   const triggerConsolidatedRaw = bodyAsJson.triggerConsolidated;
-  if (triggerConsolidatedRaw === undefined) throw new Error(`Missing triggerConsolidated`);
+  if (triggerConsolidatedRaw === undefined) return {};
   if (typeof triggerConsolidatedRaw !== "boolean") throw new Error(`Invalid triggerConsolidated`);
-
+  return { triggerConsolidatedRaw };
+}
+export function parseTriggerConsolidatedOrFail(bodyAsJson: unknown) {
+  const { triggerConsolidatedRaw } = parseTriggerConsolidated(bodyAsJson);
+  if (!triggerConsolidatedRaw) throw new Error(`Missing triggerConsolidated`);
   return { triggerConsolidatedRaw };
 }
 
 //eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function parseDisableWebhooks(bodyAsJson: any) {
   const disableWebhooksRaw = bodyAsJson.disableWebhooks;
-  if (disableWebhooksRaw === undefined) throw new Error(`Missing disableWebhooks`);
+  if (disableWebhooksRaw === undefined) return {};
```

Comment:
Fixes the parse (first lambda on patient import) fail due to optional param not being provided.